scripts/bosh

```diff
@@ -54,4 +54,4 @@ docker run  \
 	--env "BOSH_ENVIRONMENT=bosh.${SYSTEM_DNS_ZONE_NAME}" \
 	--env "BOSH_CA_CERT" \
 	--env "BOSH_DEPLOYMENT=${DEPLOY_ENV}" \
-	governmentpaas/bosh:0eff5b6a9c092f865a2b19cc4e75a3b539b82fa2 "$@"
+	governmentpaas/bosh:4012d25ceb903b46908a830b8e05773ced1c8f86 "$@"
```

Comment:
This container hasn't been updated, and is still using 3.0.1. Maybe we should update it as well so that we're using a consistent version everywhere.

Reply:
As an aside, I can't figure out where we use this script/container instead of the bosh-shell one below. It feels as though this is redundant, and could probably be removed.

Reply:
That was added by @henrytk in c1bd7cfd8cdfda8a09af117d908a39a02e7910ee, better ask him